Sources/Moya/Endpoint.swift

```diff
@@ -21,19 +21,19 @@ open class Endpoint {
     public typealias SampleResponseClosure = () -> EndpointSampleResponse
 
     /// A string representation of the URL for the request.
-    open let url: String
+    public let url: String
```

Comment:
This is a breaking change. Did we mention it in the release notes? I can't see it.

Reply:
Not a bad idea to mention it 👍

Reply:
Want me to open a PR?

Reply:
I can totally handle it

Reply:
I can do it too if you want! :)

Reply:
Feel free then lol. Just didn’t want to bother you. Want to bump the Swift version in the podspec too?

Reply:
Ah, this was because of `'let' properties are implicitly 'final'; use 'public' instead of 'open'`; it should therefore not be a breaking change.